.travis.yml

```diff
@@ -63,7 +63,11 @@ x_base_steps:
   - &deploy
     before_install:
       - travis_retry make tidy
-    install: make install
+    install:
+      # Travis image builds seem to fail on some arches (?) because this path
+      # does not exist.
+      - mkdir -p $GOPATH/bin
+      - make install
```

Comment:
I know that it is a small change but would be more assertive follow up the define guidelines here and run it be sure that all would be fine. See the steps doc here; https://sdk.operatorframework.io/docs/contribution-guidelines/developer-guide/#how-to-test-the-build-of-operator-sdk-binaries

Reply:
`make install` works fine locally and should for any system that has `go` installed. For some reason _some_ travis VM architectures do not have a `$GOPATH/bin` path when `make install` is run.

Reply:
I know that the make instal works, and yep it is just creating a dir very small. 
I just checked that it is falling now with :  

```
make[1]: Leaving directory '/home/travis/gopath/src/github.com/operator-framework/operator-sdk'
cp ./build/operator-sdk /home/travis/gopath/bin/operator-sdk
cp: cannot create regular file '/home/travis/gopath/bin/operator-sdk': No such file or directory
Makefile:58: recipe for target 'install' failed
make: *** [install] Error 1
```
The solution here makes sense, 100%. the correct step would follow the steps for we check it working and be sure that. will solve the problem. however, it shows broken now and it is too small. So, fair enough.